solidago/src/solidago/pipeline/outputs.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Literal
+
+import pandas as pd
+
+class PipelineOutput(ABC):
```

Comment:
This PipelineOutput interface, only makes sense if this object is passed to the pipeline and gets called along the way. I see that this is how it is used inside Tournesol which does not yet use the pipeline as defined by Solidago. Do I understand this correctly?

If yes, do we want to adapt the solidago's Pipeline to use solidago's PipelineOutput?

Reply:
`Pipeline.__call__()` now accepts an optional `output` to customize the output and write scores during the pipeline execution:
https://github.com/tournesol-app/tournesol/blob/631257dcb3dc7608892a5c59fe21c40e34033460/solidago/src/solidago/pipeline/pipeline.py#L122-L131